lib/util/cmcd_manager.js

```diff
@@ -8,12 +8,15 @@ goog.provide('shaka.util.CmcdManager');
 
 goog.require('goog.Uri');
 goog.require('shaka.log');
-
+goog.require('shaka.net.NetworkingEngine');
+goog.require('shaka.util.IDestroyable');
 
 /**
  * @summary
  * A CmcdManager maintains CMCD state as well as a collection of utility
  * functions.
+ *
+ * @implements {shaka.util.IDestroyable}
```

Comment:
IReleasable is for synchronous tear-down.  You could use that instead here, I think.  And since this is an internal class, if we need to go back to IDestroyable (async) at a later time, it doesn't hurt compatibility.

Reply:
Refactored to use `IReleasable`